libs/langgraph/src/pregel/algo.ts

```diff
@@ -143,7 +189,10 @@ export function _applyWrites<Cc extends Record<string, BaseChannel>>(
   tasks: WritesProtocol<keyof Cc>[],
   // eslint-disable-next-line @typescript-eslint/no-explicit-any
   getNextVersion?: (version: any, channel: BaseChannel) => any
-): void {
+): Record<string, PendingWriteValue[]> {
+  const filteredChannels = Object.fromEntries(
```

Comment:
See above, I *think* this is filtering out managed values but more clarity around naming would be nice.

Reply:
it is, will rename